### PR TITLE
Add: Update feature to User Guide

### PR DESCRIPTION
**Pull request by rmj1405**
Added the description of update feature into user guide
Added use cases of update feature into developer guide

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -307,7 +307,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
   * 4a2. WIFE asks if the user wish to add the new tags into the tag list.
     Use case resumes at step 2.
 
+**Use case: Update a food item**
```
Comment (Bisceto) [inline docs/DeveloperGuide.md:310]: Can update the use case to follow the numbering of the use cases!

**Review by Bisceto — COMMENTED** [review]
Need to update the case numbering

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (rmj1405) [reply docs/DeveloperGuide.md]: rectified!

**Review by Bisceto — APPROVED** [review]
LGTM!

**Pull request merged**